tsconfig.base.json

```diff
@@ -1,6 +1,7 @@
 {
 	"extends": "@sapphire/ts-config",
 	"compilerOptions": {
+		"target": "ES2020",
```

Comment:
Should be in ts-config imo. @Favna thoughts?

Reply:
Depends. Do we want to support maintenance LTS versions of NodeJS with the tsconfig?
If so then no, because Node 12 won't go EOL until 2022-04-30 and it doesn't support ES2020. 
If not then the there is just 1 ES2020 feature that is not available in Node 14, but considering it wasn't even in 16.0 I think it might still get backported.

![image](https://user-images.githubusercontent.com/4019718/126239722-cec5934c-e498-452a-9384-07d964e28dff.png)

Reply:
Maintenance should practically be dead, so.. I'm game for ES2020 support

Reply:
Besides, discord.js v13 requires Node.js v14.

Reply:
Updated @sapphire/ts-config to v3.0.0. Updated dependency and then remove the target field from this config.